README.md

```diff
@@ -36,6 +36,7 @@ Trust Wallet Core is a cross-platform library that implements low-level cryptogr
 <a href="https://kinecosystem.org/" target="_blank"><img src="https://raw.githubusercontent.com/TrustWallet/tokens/master/coins/2017.png" width="32" /></a>
 <a href="https://decred.org/" target="_blank"><img src="https://raw.githubusercontent.com/TrustWallet/tokens/master/coins/42.png" width="32" /></a>
 <a href="https://www.thetatoken.org/" target="_blank"><img src="https://raw.githubusercontent.com/TrustWallet/tokens/master/coins/500.png" width="32" /></a>
+<a href="https://luxcore.io/" target="_blank"><img src="PLACE_HOLDER" width="32" /></a>
```

Comment:
Please add actual image.

Reply:
My image is in the Token PR, should I upload it somewhere else while I wait for the SatoshiLab Maintainers to add LUX?
The PR: https://github.com/TrustWallet/tokens/pull/1358

Reply:
For now use the actual name it is going to end up being.

Reply:
Thanks. Which name?